database-mongodb/index.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+const mongoUri = 'mongodb://localhost/fec_ratings';
+
+mongoose.connect(mongoUri, { useNewUrlParser: true }); 
+const db = mongoose.connection; 
+
+db.once('open', () => {
+    console.log('Now connected to MongoDB called fec_ratings...');
+})
+
+module.exports = db;
```

Comment:
Good separation of models into their own files. Consider checking for errors on db connection.